examples/cluster-configs/trn1-16-nodes-pcluster.md

```diff
@@ -92,10 +98,16 @@ Where
 
 `suppress-validators` is used here to generalize this command so it will not run into error triggered by tagging policies, if any.
 
-This will create a ParallelCluster in your AWS account, and you may inspect the progress in AWS CloudFormation console.
+This will create a ParallelCluster in your AWS account, and you may inspect the progress in AWS CloudFormation console. 
 
+You may also check cluster status using `pcluster` command, for example: 
 
-3. After the cluster is created successfully, perform the [post-install actions](../general/ami/post_installation.md) to configure the head node and compute nodes. 
+`pcluster describe-cluster -r us-west-2 -n My-ParallelCluster-Trn1`
+
+
+3. After the cluster is created successfully, post-install actions happen as a part of the cluster creating process. Post-install actions now takes place automatically via `CustomActions` indicated in `launch.yaml` to configure the head node and any static compute nodes (`MinCount` > 0). For your information, `CustomActions` will install Neuron drivers and runtime, EFA drivers (if multi-instance training), and Neuron tools. 
```

Comment:
Remove "For your information".

Remove "(if multi-instance training)"

Reply:
Remove the first sentence and do ""During the cluster creation process, post-install actions take place automatically ..."

Reply:
Done. See https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/14/commits/3f5c5a96f33cb97b42d06f42d669a749f9aca161